src/test/resources/features/SubmitUKTR.feature

```diff
@@ -1,4 +1,3 @@
-@apiTests
```

Comment:
is this file still needed? If not, please delete it

Reply:
Created ticket -PIL-1516 to refactor